server/config/definition_unix.go

```diff
@@ -36,6 +36,7 @@ type Config struct {
 	}
 	Secret Bytes
 	SMTP   struct {
+		Authtype string `default:"LOGIN"`
```

Comment:
Could you add the possible options and their implications to the documentation site still? https://github.com/offen/offen/blob/development/docs/docs/running-offen/configuring-the-application.md is probably the location for that.

Reply:
Added some documentation. Does this suffice or should I go more into detail?